lib/workers/repository/onboarding/branch/index.spec.ts

```diff
@@ -71,8 +71,10 @@ describe('workers/repository/onboarding/branch/index', () => {
       git.getFileList.mockResolvedValue(['package.json']);
       fs.readLocalFile.mockResolvedValue('{}');
       await checkOnboardingBranch(config);
+      const file = git.commitFiles.mock.calls[0][0].files[0];
```

Comment:
```suggestion
      const file = git.commitFiles.mock.calls[0][0].files[0] as FileAddition;
```

We know it can't be something else here